adjust_dataset.py

```diff
@@ -0,0 +1,26 @@
+#Script changing labels of emotions to 1 - smile, 0 - no smile and saving changed dataset to new csv file
+
+
+def main():
+    import pandas as pd
```

Comment:
I also mentioned already - imports should be declared at the beginning of the file for clarity, not within the function.